common/session/class.SessionManager.php

```
@@ -114,4 +118,47 @@ public static function isAnonymous()
     }
+
+    public static function extractAccessTokenFromRequest(): string
+    {
+        $authorizationHeader = explode(' ', $_SERVER['HTTP_AUTHORIZATION'] ?? '', 2);
+        return array_pop($authorizationHeader);
+    }
+
+    public static function parseAccessToken(string $accessToken): ?array
+    {
+        /** @noinspection PhpUnusedLocalVariableInspection */
+        @[$_, $payload] = explode('.', $accessToken);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$_'. (undefined)

(UnusedLocalVariable)